src/test/scala/uk/gov/hmrc/api/service/AuthService.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ */
+
+package uk.gov.hmrc.api.service
+
+import play.api.libs.ws.StandaloneWSRequest
+import uk.gov.hmrc.api.client.HttpClient
+import uk.gov.hmrc.api.conf.TestConfiguration
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class AuthService extends HttpClient {
+  val host: String = TestConfiguration.url("auth")
+
+  val authPayload: String =
```

Comment:
This payload is for another service. We need a payload for our service.

Reply:
Replaced with proper payload